x-pack/plugins/cases/server/client/cases/bulk_update.ts

```diff
@@ -309,7 +309,7 @@ export interface UpdateRequestWithOriginalCase {
  *
  * @ignore
  */
-export const update = async (
+export const bulkUpdate = async (
```

Comment:
I renamed the method of the case client from `update` to `bulkUpdate` to make it clearer what the function does.